src/main/java/duke/Commands/AddInput.java

```diff
@@ -0,0 +1,106 @@
+package duke.Commands;
+
+import duke.Exceptions.*;
```

Comment:
It is required in the style guide to list out imported classes explicitly instead of using wildcard.